ehealthid-rp/src/main/java/com/oviva/ehealthid/relyingparty/Main.java

```diff
@@ -136,20 +141,29 @@ public void start() throws ExecutionException, InterruptedException {
 
     server =
         SeBootstrap.start(
-                new App(
-                    config, keyStore, tokenIssuer, clientAuthenticator, meterRegistry, authService),
+                new App(config, keyStore, tokenIssuer, clientAuthenticator, authService),
                 Configuration.builder().host(config.host()).port(config.port()).build())
             .toCompletableFuture()
             .get();
 
     var localUri = server.configuration().baseUri();
     logger.atInfo().log("Magic at {} ({})", config.baseUri(), localUri);
+
+    managementServer =
+        SeBootstrap.start(
+                new ManagementApp(meterRegistry),
+                Configuration.builder().host(config.host()).port(config.managementPort()).build())
+            .toCompletableFuture()
+            .get();
```

Comment:
If you only need to serve bare http, can you maybe just use the underlying undertow server? You probably don't need all the JAX-RS bits inbetween.
Here's an example: https://github.com/oviva-ag/outbox-relay/blob/807c8ec51d9eabf0285c1d861663ed11d44367d3/service/src/main/java/com/oviva/outboxrelay/http/HttpServer.java#L37